### DISCUSSION TITLE
Group Assignment #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#31] [Assignments]
# Survey Diagram and Website Wire-frame
## Due 5/11

Have a group member comment on this following thread with the following:
- Link or photos for the survey diagram 
- Link or photos for the website wire-frame.

**Comment by emmashanson** [comment]
Link for survey diagram: https://www.figma.com/file/8elFmjnRkJcfbxqMB1HvK2/aa191-survey-diagram?node-id=0%3A1
wireframe: https://drive.google.com/file/d/1ildstZwhVObZHEVEwlyE0eOEdJ1j3Eou/view?usp=sharing

**Reply by angelineeow** [reply]
Group 8: 
Survey diagram & wireframe:
[Note_May_10_2022.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8666416/Note_May_10_2022.pdf)
[Group8.drawio.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8666417/Group8.drawio.pdf)

**Reply by albertkun** [reply]
> Link for survey diagram: https://www.figma.com/file/8elFmjnRkJcfbxqMB1HvK2/aa191-survey-diagram?node-id=0%3A1 wireframe: https://drive.google.com/file/d/1ildstZwhVObZHEVEwlyE0eOEdJ1j3Eou/view?usp=sharing

Hello Emma! I need access to the wireframe sadly! :(

**Comment by ananyakodali17** [comment]
Jenny & Ananya
Survey diagram: 
[group assignment flow diagram.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8666570/group.assignment.flow.diagram.pdf)

Wireframe:
[wireframe 191a.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8666567/wireframe.191a.pdf)

**Comment by NishuKrish** [comment]
Nishaanth and Ivy
Survey Diagram: https://docs.google.com/drawings/d/1YZW0f_n6rt-9vNPdgO0-5ZPhsFYCP7t8bj0DQK8N97w/edit?usp=sharing
Wireframe: https://drive.google.com/file/d/1-saLx9h-5TGMiW9HMugkphiLAq2-R5Rw/view?usp=sharing

**Comment by codingdianat** [comment]
Diana and Steven:
Survey Diagram: https://www.figma.com/file/N0n4zIPqIV3bZATLImlkHG/Chinese-Discrimination-Asia-Am?node-id=0%3A1
Wireframe: https://www.figma.com/file/38e4RKebaHbrraL06Oh0F0/Chinese-Discrimination%40UCLA-Wireframe?node-id=0%3A1

**Comment by ashley-lu** [comment]
Ashley, Haven, and Shubham:
Survey Diagram: https://drive.google.com/file/d/14rJsdA-4eMqUK_rha7HMAac21QxnuaDj/view?usp=sharing
Wireframe: https://www.figma.com/file/uh1hVasQvGdcqb0XkMjVaS/Untitled?node-id=0%3A1

**Comment by penelopelamb** [comment]
Sine, Alexis, Lillan
Survey diagram: https://drive.google.com/file/d/1rBYGhGMA4k1E2didbfLAV1FJ9zJXr2aU/view?usp=sharing
Wireframe: https://drive.google.com/file/d/13Fb7JHeGU-5QA8qHogDnqNMnsFb7fP8t/view?usp=sharing

**Comment by willa-21** [comment]
Ngoc, Xiong, & Willa -- https://www.canva.com/design/DAFAaD-vci0/DKxGpf2-iSIFisfRApAWwQ/view?utm_content=DAFAaD-vci0&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by antonyblin** [comment]
Kaitlyn, Antony
Survey Diagram: [Untitled_Artwork.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8675478/Untitled_Artwork.pdf)
Wireframe: [Note May 11, 2022.pdf](https://github.com/albertkun/22S-ASIAAM-191A/files/8675475/Note.May.11.2022.pdf)
